app/packages/utilities/src/fetch.ts

```
@@ -408,3 +430,18 @@
       );
     });
 };
+
+const getFirstFilterPath = (requestBody: any) => {
+  if (requestBody && requestBody.query) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)